frontend/apps/filemanager/filemanagerbookinfo.lua

```diff
@@ -43,13 +43,27 @@ function BookInfo:show(file, book_props)
 
     local directory, filename = util.splitFilePathName(file)
     local filename_without_suffix, filetype = util.splitFileNameSuffix(filename) -- luacheck: no unused
+    if filetype:lower() == "zip" then
+        local filename_without_sub_suffix, sub_filetype = util.splitFileNameSuffix(filename_without_suffix) -- luacheck: no unused
+        sub_filetype = sub_filetype:lower()
+        local supported_sub_filetypes = { "fb2", "htm", "html", "log", "md", "txt" }
+
+        for __, typ in ipairs(supported_sub_filetypes) do
```

Comment:
typ or type?

Reply:
"type" on first intention, but it got highlighted by vim, as `type("abc")` is a lua builtin function.
(It works with "type", but it feels strange to have it highlighted.)
So, I went for `typ`. I can use `t` if we'd prefer to not have orthograph errors :)

Reply:
I think I prefer t, lol (but whichever)